docs/client-generic.md

```diff
@@ -1,4 +1,4 @@
-# Client installation
+# Generic client setup
```

Comment:
I think this is a Linux client setup. There are two different kinds of Linux clients:
* Linux servers (w/ raw strongSwan)
* Linux desktops (w/ Network Manager)

Let's call this client-linux.md and "Linux client setup" and I'll call the other one client-networkmanager.md and "Network Manager client setup"?